src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -6631,11 +6633,11 @@ namespace HeatBalanceManager {
                                                                      state.dataIPShortCut->lAlphaFieldBlanks,
                                                                      state.dataIPShortCut->cAlphaFieldNames,
                                                                      state.dataIPShortCut->cNumericFieldNames);
-            if (UtilityRoutines::IsNameEmpty(state, FrameDividerNames(1), state.dataHeatBalMgr->CurrentModuleObject, ErrorsFound)) continue;
+            if (UtilityRoutines::IsNameEmpty(state, FrameDividerAlphas(1), state.dataHeatBalMgr->CurrentModuleObject, ErrorsFound)) continue;
 
             // Load the frame/divider derived type from the input data.
             ++FrameDividerNum;
-            state.dataSurface->FrameDivider(FrameDividerNum).Name = FrameDividerNames(1);
```

Comment:
Can use an `auto &frameDivider = state.dataSurface->FrameDivider(FrameDividerNum);` shortcut to tidy up this code a bit. I understand you don't have many long lines here, but still ...

Reply:
Good idea